services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -16,7 +16,8 @@
 
 public class RouteOptionsTest {
 
-  private static final String ROUTE_OPTIONS_JSON = "{\"baseUrl\":\"base_url\",\"user\":\"user\",\"profile\":\"profile\",\"coordinates\":[[1.0,2.0],[3.0,4.0]],\"access_token\":\"token\",\"uuid\":\"requestUuid\",\"waypoint_targets\":\";33,44;55,66\"}";
+  private static final String ROUTE_OPTIONS_JSON =
```

Comment:
I'm missing the `exclude` option https://docs.mapbox.com/api/navigation/#retrieve-directions

Reply:
fixed